src/tests/multihost/ad/test_adparameters.py

```diff
@@ -692,8 +692,10 @@ def test_0017_gssspnego_adjoin(self, multihost):
         multihost.client[0].run_command(remove_pcap)
         assert status == 'PASS'
 
+    @staticmethod
```

Comment:
Do we need staticmethod decorator here? @pytest.mark.skip should just skip this test even if it is picked because of the tier1 marker right?

Reply:
The static analysis complains about unused self and suggesting the conversion to static method.